coral/src/app/features/topics/request/TopicRequest.tsx

```diff
@@ -178,16 +170,16 @@ function TopicRequest() {
             <Box paddingY={"l1"}>
               <Divider />
             </Box>
-            <Box component={Flexbox} gap={"l1"}>
-              <Box component={FlexboxItem} grow={1} width={"1/2"}>
+            <Box component={Box.Flex} gap={"l1"}>
+              <Box component={Box.Flex} grow={1} width={"1/2"}>
```

Comment:
Same comment as above, this is what this markup renders:

<img width="860" alt="Screenshot 2023-09-29 at 09 22 02" src="https://github.com/Aiven-Open/klaw/assets/20607294/171d1cd5-65a9-4ac1-85be-4f0a892b9683">

This should be used insterad to preserve original layout

```tsx
<Box.Flex gap={"l1"}>
  <Box width={"1/2"}>
    <Textarea<Schema>
      name="description"
      labelText="Topic description"
      rows={5}
      required={true}
    />
  </Box>
  <Box width={"1/2"}>
    <Textarea<Schema>
      name="remarks"
      labelText="Message for approval"
      placeholder="Comments about this request for the approver."
      rows={5}
    />
  </Box>
</Box.Flex>
```


Renders:
<img width="967" alt="Screenshot 2023-09-29 at 09 21 00" src="https://github.com/Aiven-Open/klaw/assets/20607294/4e197e71-12c3-4d9b-95ee-490498808eea">